### ISSUE TITLE
Cannot open links anymore

### ISSUE THREAD

**Issue by dschrempf** [jkitchin/org-ref#726]
Hello, I get the following error when opening org ref links:
```
Debugger entered--Lisp error: (wrong-type-argument stringp nil)
  insert-file-contents(nil)
  org-ref-cite-candidates()
  org-ref-cite-click-helm(nil)
  funcall(org-ref-cite-click-helm nil)
  (lambda (_) (funcall org-ref-cite-onclick-function nil))("CITATIONKEY")
  org-link-open((link (:type "autocite" :path "CITATIONKEY" :format plain :raw-link "autocite:CITATIONKEY" :application nil :search-option nil :begin 338 :end 356 :contents-begin nil :contents-end nil :post-blank 0 :parent (paragraph (:begin 315 :end 358 :contents-begin 315 :contents-end 357 :post-blank 1 :post-affiliated 315 :parent nil)))) nil)
  org-open-at-point(nil)
  funcall-interactively(org-open-at-point nil)
  call-interactively(org-open-at-point nil nil)
  command-execute(org-open-at-point)
```
Do you think this is `org-ref` related, or a problem of my configuration?

EDIT: I should add that opening links has always worked for me, and this must have been a recent Emacs/Org-Mode/Org-Ref update.

**Comment by jkitchin** [comment]
I cant think of a reason this would be an org-ref problem, but there isn't enough information to say. Can you provide a small working example?

**Comment by dschrempf** [comment]
OK, so I nailed it down to the fact that the link to the bibliography needs to
be present in the file. So with

```
Hello.
autocite:Trump2020
```

I get the error. But with

```
#+latex_header: \addbibresource{~/path/to/bibliography.bib}

Hello.
autocite:Zwaenepoel2019
```

Everything works OK.

Is this expected behavior? I thought that org ref is independent of LaTeX and
should also work without this header line? Or maybe my setup screws things up?

Thanks,
Dominik

**Comment by jkitchin** [comment]
It could be that you don't have a default bibliography setup correctly to find the citations in. 

org-ref needs a way to find a bibliography file that has the citation keys in it, and it first looks in the org file for a latex command or bibliography link, then looks in the default bibliography variable (which is a list of bibtex files to use).

**Comment by dschrempf** [comment]
OK, I found the error. The default bibliography was a string, not a list (which would contain one string in my case). It is a little weird that no error is thrown in this case. I just saw that bibtex has a good way of handling this: the `bibtex-completion-bibliography` variable can be a single string or a list of strings. Maybe you want to copy this behavior? Anyways, I close this issue, thanks a lot for your help.
